chain/client/src/client.rs

```diff
@@ -2448,6 +2542,23 @@ impl Client {
         Ok(false)
     }
 
+    fn set_state_sync_hash<'a>(
+        mut update: ChainStoreUpdate<'a>,
```

Comment:
Can this be a method on ChainStoreUpdate?

Reply:
just removed it